scheds/rust/scx_layered/src/main.rs

```diff
@@ -1888,9 +1888,14 @@ impl<'a> Scheduler<'a> {
             layers.push(Layer::new(spec, &topo, growth_order)?);
         }
 
-        let idle_qos_enabled = layers
+        let mut idle_qos_enabled = layers
             .iter()
             .any(|layer| layer.kind.common().idle_resume_us.unwrap_or(0) > 0);
+        if !cpu_idle_resume_latency_supported() && idle_qos_enabled {
```

Comment:
nit: this and is backwards. no need to check the file exists if you don't need it.

Reply:
Ahh good catch!